core/unix/signal.c

```diff
@@ -3752,6 +3752,29 @@ handle_client_action_from_cache(dcontext_t *dcontext, int sig, dr_signal_action_
     return true;
 }
 
+static bool
+handle_sent_to_client_signal(dcontext_t *dcontext, int sig, sigframe_rt_t *frame,
```

Comment:
nit: The function name sounds like the signal was already sent to the client before this function was called

Reply:
I guess it is a bit misleading. Would `send_signal_to_client_and_handle_action()` be better? I am terrible with function names.

Reply:
`send_signal_to_client_and_handle_action()` SGTM